### DISCUSSION TITLE
oauth2 docstrings are out of date

### DISCUSSION THREAD

**Discussion by dimaqq** [fastapi/fastapi#8436] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
async def token(form_data: OAuth2PasswordRequestForm = Depends()):                      
    data = form_data # .parse()
```

### Description

https://github.com/tiangolo/fastapi/blob/97f04ab58c2c047d6d2ff7d803890a90190109c9/fastapi/security/oauth2.py#L17-L20

Maybe in the past this was the case, but today there's no `.parse` on `form_data`

Some 3p online resource omits the `.parse()` and accessing fields directly on `form_data` works, so presumably that's the correct way now?

### Operating System

Linux

### FastAPI Version

0.88.0

### Python Version

3.10

**Comment by nebiyuelias1** [comment]
I created a pr for this: https://github.com/tiangolo/fastapi/pull/5772
